ISSUE.md

```diff
@@ -0,0 +1,56 @@
+# Issue reporting on Podman
+
+The Podman team cares for our users and communities.  When someone has a problem with
+Podman and takes the time to report an [issue](https://github.com/containers/podman/issues)
+on our Github, we deeply appreciate the effort.  We want to help. Consider reading
+our [support](SUPPORT.md) document prior to submitting an issue as well.
+
+## Considerations when reporting an issue upstream
+
+### Where to report your issue
+
+If you are running Podman acquired from a Linux distribution and that Linux distribution has a
+bug reporting mechanism, then please report the bug there.
+
+### Development or latest version
+
+We view this Github repository as an upstream repository for development
+and the latest Podman version.
+
+The term "latest version" refers to our mainline development tree or the
+[latest release](https://github.com/containers/podman/releases/latest).
+
+You are welcome to replicate the bug with upstream code and report that.
+
+### Bugs vs features
+
+A bug is when something in Podman is not working as it should or has been described.  A
+feature or enhancement is when you would like Podman to behave differently.
```

Comment:
Should probably mention that we accept issues for both, but features are much less likely to be worked on by maintainers.